### ISSUE TITLE
get_coauthors: normalize authors names

### ISSUE THREAD

**Issue by DominiqueMakowski** [jkeirstead/scholar#58]
@cimentadaj A minor suggestion: I've noticed some people have their name written in full uppercase (Serge NICOLAS) while some others don't (Dominique Makowski). 

Wouldn't it be more neat-looking (especially in the network graph) to homogenise the author names? 

It can be easily done by title-casing the names vector: 

```r
stringr::str_to_title("Serge NICOLAS")
stringr::str_to_title("Dominique Makowski")
``` 
What do you think?

**Comment by cimentadaj** [comment]
Yes, this is a good idea. Implemented in #57 . Thanks!

**Comment by mkiang** [comment]
Just FYI, if you want to do the same thing but don't want to depend on `stringr` for a single function, this is built into R: `tools::toTitleCase()`

**Comment by cimentadaj** [comment]
Yeah, good point, However `stringr` is already a depedency in `scholar` so no problem.
